api-tester/src/main/java/com/revenuecat/apitester/java/StoreProductAPI.java

```diff
@@ -2,21 +2,28 @@
 
 import com.android.billingclient.api.ProductDetails;
 import com.revenuecat.purchases.ProductType;
+import com.revenuecat.purchases.StoreProductExtensionsKt;
+import com.revenuecat.purchases.models.GooglePurchaseOption;
 import com.revenuecat.purchases.models.GoogleStoreProduct;
+import com.revenuecat.purchases.models.GoogleStoreProductKt;
 import com.revenuecat.purchases.models.Price;
+import com.revenuecat.purchases.models.PurchaseOption;
 import com.revenuecat.purchases.models.StoreProduct;
 
-import org.json.JSONObject;
+import java.util.List;
 
 @SuppressWarnings({"unused"})
 final class StoreProductAPI {
     static void check(final StoreProduct product) {
-        final String sku = product.getSku(); //TODOBC5 rename
+        final String productId = product.getProductId();
+        final String sku = product.getSku();
         final ProductType type = product.getType();
         final Price price = product.getOneTimeProductPrice();
         final String title = product.getTitle();
         final String description = product.getDescription();
         final String subscriptionPeriod = product.getSubscriptionPeriod();
+        List<PurchaseOption> purchaseOptions = product.getPurchaseOptions();
+        GoogleStoreProduct underlyingProduct = GoogleStoreProductKt.getGoogleProduct(product);
```

Comment:
Hmm this is interesting... Might be useful to find an easier way to do this in Java... But we can do that later

Reply:
yeah would be for all extensions...

Reply:
@tonidero i know this is a very old comment, but i'm trying to clean up some TODOs... did you have any ideas for how to better expose extension functionsin java?

Reply:
we could do something like this: https://github.com/RevenueCat/purchases-android/pull/727#pullrequestreview-1244206533 maybe?

Reply:
Hmm I think what you posted should work yeah, so I would probably go with this first.

An alternative would be creating a helper class to handle the conversion. I would choose this if we wanted to avoid static methods which make it a harder to test API... But considering the simple logic this actually has, I think it's ok to have a static method.